### ISSUE TITLE
Until Tensorflow 2.0, there had been an instance norm layer in tf.contrib.layers - please bring it back

### ISSUE THREAD

**Issue by feature-engineer** [tensorflow/tensorflow#33080]
<em>Please make sure that this is a feature request. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:feature_template</em>

**System information**
- TensorFlow version: 2.0:
- Are you willing to contribute it: Yes:

There's already the code for this layer in tensorflow 1.14, and (with a bit less parameter choice) in pix2pix example in tensorflow 2.0, It's trivial to add this layer to tensorflow.keras.layers, and I'm willing to do it.

**Describe the feature and the current behavior/state.**
Instance norm was found to be[ more effective](https://arxiv.org/abs/1607.08022) than any other form of normalization for convolutional neural networks with small batches.

It is used in tensorflow's [official example for pix2pix ](https://www.tensorflow.org/tutorials/generative/pix2pix), and was present in tf.contrib.layers in [tensorflow 1.14](https://www.tensorflow.org/versions/r1.14/api_docs/python/tf/contrib/layers/instance_norm).

Instance norm is normalizes each channel according to the channel's spatial mean and standard deviation. 

**Will this change the current api? How?**
Add tensorflow.keras.layers.InstanceNormalization

**Who will benefit with this feature?**
Anyone who's working on a CNN with small batches.

**Comment by WindQAQ** [comment]
Hello, instance normalization layer has been moved to tensorflow-addons.
https://github.com/tensorflow/addons/blob/master/tensorflow_addons/layers/normalizations.py#L277

**Comment by gigadeplex** [comment]
@WindQAQ does not support Windows...

**Comment by rmothukuru** [comment]
@feature-engineer,
Can you please let us know if @WindQAQ's comment has answered your query. If not, can you please give more details. Thanks!

**Comment by feature-engineer** [comment]
@rmothukuru If tensorflow-addons was working on Windows, it would have been a viable solution. As it stands, windows users of tensorflow face a regression from the 1.14 version.

**Comment by gowthamkpr** [comment]
@feature-engineer Is this https://github.com/tensorflow/addons/blob/master/tensorflow_addons/layers/normalizations.py#L277

not working on windows?

**Comment by feature-engineer** [comment]
@gowthamkpr Not yet:

https://github.com/tensorflow/addons/issues/173

**Comment by amitport** [comment]
@gowthamkpr 
besides `tf.contrib.layers` everything that was moved to `tensorflow-addons` causes a blocking regression for people that used it on windows and would otherwise move to `tf@2`

**Comment by gowthamkpr** [comment]
@amitport @feature-engineer Thanks for the report. Will  keep you updated on this part.

**Comment by gowthamkpr** [comment]
@seanpmorgan Any updates on the support to windows. Thanks!

**Comment by seanpmorgan** [comment]
> @seanpmorgan Any updates on the support to windows. Thanks!

Hi @gowthamkpr please follow this issue for updates: https://github.com/tensorflow/custom-op/issues/24

There has been a to-do list proposed to make windows custom-ops work.. just waiting for the issue to be complete and then addons will follow suit.

**Comment by gowthamkpr** [comment]
I am  closing this issue as of now as it has been tracked already. Please follow the issue mentioned by Sean and we will update you as soon as the work is done. Thanks!
